src/components/aboutModal/aboutModal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { AboutModal as PfAboutModal, TextContent, TextList, TextListItem } from '@patternfly/react-core';
+import { detect } from 'detect-browser';
+import moment from 'moment/moment';
+import { helpers } from '../../helpers';
+import { useUserApi } from '../../hooks/useLoginApi';
+import { useStatusApi, type ApiStatusSuccessType } from '../../hooks/useStatusApi';
+import backgroundImageSrc from '../../images/aboutBg.png';
+
+interface AboutModalProps {
+  currentYear?: string;
+  isOpen?: boolean;
+  onClose?: () => void;
+  titleImg?: string;
+  uiName?: string;
+  uiVersion?: string;
+  useStatus?: typeof useStatusApi;
+  useUser?: typeof useUserApi;
+}
+
+const AboutModal: React.FC<AboutModalProps> = ({
+  currentYear = moment.utc(helpers.getCurrentDate()).format('YYYY'),
```

Comment:
should we move from moment lib since now it is considered a legacy project ? I am not sure what we should use instead, but I thought it was worth it to bring the idea here. From a fast search online libs date-fns or dayjs pop up, but I also don't know if supported to downstream builds.